assets/js/app/patience-is-a-virtue.js

```diff
@@ -18,7 +26,7 @@ $('*[data-patience]').on('click', function() {
 });
 
 window.reEnablePatientButtons = function() {
-  $('*[data-patience]').each(function() {
+  $('*[data-original-class]').each(function() {
```

Comment:
what is data-original-class?

Reply:
It gets set on line 20, when a button (or something else) gets a spinner.